indigo_api/data_migrations/__init__.py

```diff
@@ -72,3 +72,26 @@ def strip_whitespace(self, xml):
 
         return xml
 
+
+class UpdateAKNNamespace(AKNMigration):
+    """ Update all instances of:
+    <akomaNtoso xmlns="http://www.akomantoso.org/2.0" xsi:schemaLocation="http://www.akomantoso.org/2.0 akomantoso20.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    to
+    <akomaNtoso xmlns="http://docs.oasis-open.org/legaldocml/ns/akn/3.0">
+    """
+    def update_namespace(self, xml):
+        if "http://www.akomantoso.org/2.0" in xml[:300]:
```

Comment:
this isn't strictly necessary since the replace is a no-op if it can't find the string.

Reply:
I know it's _highly_ unlikely, but don't we want to guard against replacing it outside of the first line?

Reply:
> > This PR also needs to tell slaw to use AKN3 namespaces
> 
> the Gemfile already specifies gem 'slaw', git: 'https://github.com/longhotsummer/slaw', branch: 'akn3'
> is there something else that needs to happen?

Slaw now does AKN3 by default.